### PR TITLE
quarkus example app using auth for registry and kafka

### PR DESCRIPTION
**Pull request by famartinrh**

### CODE REVIEW

**Comment by EricWittmann** [comment]
Did you intend to include the vscode and factorypath files?

**Comment by famartinrh** [comment]
no I did not, fixed

File: quarkus-auth/src/main/java/io/apicurio/example/schema/avro/Event.java [hunk 1]
```diff
@@ -0,0 +1,488 @@
...
+
+import org.apache.avro.generic.GenericArray;
+import org.apache.avro.specific.SpecificData;
+import org.apache.avro.util.Utf8;
+import org.apache.avro.message.BinaryMessageEncoder;
+import org.apache.avro.message.BinaryMessageDecoder;
+import org.apache.avro.message.SchemaStore;
+
+/** Avro Schema for Event */
+@org.apache.avro.specific.AvroGenerated
```
Comment (carlesarnal) [inline quarkus-auth/src/main/java/io/apicurio/example/schema/avro/Event.java:16]: Do we want to include the class generated by Avro?

**Review by carlesarnal — APPROVED** [review]
Besides the minor question about Avro generated classes, LGTM.

**Pull request merged**